### PR TITLE
Sub-user Management UI

### PR DESCRIPTION
**Pull request by mayethsc**

### CODE REVIEW

File: application/modules/hirer/views/user.php [hunk 1]
```diff
@@ -1,10 +1,262 @@
...
+    array(
+        'name' => 'update-access',
+        'id' => 'update-access',
+        'value' => 'Save',
+        'class' => 'btn btn-success border-0 ml-md-auto mb-3'
+    )
+);
+
+// Draft Info Content
+$infoText = '<div class="col-11">All the additional users have been deactivated, they will not be able to access your WorkAbroad account. To activate it, please call 798-5610.  <strong><em>Please call 798-5610 to get additional users</em></strong>.';
```
Comment (princeryansy) [inline application/modules/hirer/views/user.php]: Kindly use our defined constant for the office phone number and use lang for static contents.

File: application/modules/hirer/views/user.php [hunk 2]
```diff
@@ -1,10 +1,262 @@
...
+                $this->table->add_row(
+                    array(
+                        $i . '.',
+                        $value['name'],
+                        $value['email'],
+                        $value['mobile'],
+                        $value['login_id'],
+                        '<div class="custom-control custom-checkbox">
+                            ' . $postJobInput . '
+                            <label class="custom-control-label" for="job-posting-' . $i . '"> Job Posting</label>
```
Comment (princeryansy) [inline application/modules/hirer/views/user.php]: Add these labels in the lang files. apply to similar occurences

File: application/modules/hirer/views/user.php [hunk 3]
```diff
@@ -1,10 +1,262 @@
...
+         </div>
+         
+         <?php
+            echo form_close();
+     }
+    ?>
+    </div>
+
+    <div class="card shadow mb-4 p-4">
+        <p class="h5 text-danger">Inactive</p>
```
Comment (princeryansy) [inline application/modules/hirer/views/user.php]: add static content in lang files.

File: application/modules/hirer/views/user.php [hunk 4]
```diff
@@ -1,10 +1,262 @@
...
+                        <label class="custom-control-label" for="status-inactive-' . $a . '"> Set to Active </label>
+                    </div>
+
+                    ',
+                    '<a href="#" class="btn btn-warning btn-xs rounded"><i class="simple-icon-note"></i></a>'
+                    )
+                );
+            }
+        } else {
+            $noInactive = array('data' => 'No inactive user found.', 'class' => 'align-middle text-center', 'colspan' => 12);
```
Comment (princeryansy) [inline application/modules/hirer/views/user.php]: add static content in lang file.

**Comment by princeryansy** [comment]
Good usage of the CI table library. Address all static content issues.

File: application/config/config_table.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+$config['users'] = array(
+        'table_open' => '<table class="table table-sm table-hover table-responsive-lg">',
```
Comment (princeryansy) [inline application/config/config_table.php]: reduce tab spaces to 4, apply to similar occurrences below.

File: application/helpers/WA_form_helper.php [hunk 6]
```diff
@@ -153,3 +153,14 @@ function set_value($field, $default = '', $html_escape = false)
 
     return ($html_escape) ? html_escape($value) : $value;
 }
+
+function populateFormData($fields = array(), $data = array()) {
```
Comment (princeryansy) [inline application/helpers/WA_form_helper.php]: function opening brackets should be on a new line

File: application/helpers/WA_form_helper.php [hunk 7]
```diff
@@ -153,3 +153,14 @@ function set_value($field, $default = '', $html_escape = false)
...
+
```
Comment (princeryansy) [inline application/helpers/WA_form_helper.php]: there should be no extra line after a function declaration.

File: application/modules/hirer/controllers/Users.php [hunk 8]
```diff
@@ -116,6 +111,9 @@ public function additional()
             'status' => 0
         );
 
+        $this->initTableSettings();
+        $this->table->set_heading(self::TABLE_HEADERS);
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Users.php]: remove set_heading

**Comment by mayethsc** [comment]
Approved already by @princeryansy .

**Pull request merged**